integrated_channels/integrated_channel/models.py

```diff
@@ -1,13 +1,15 @@
 """
 Database models for Enterprise Integrated Channel.
 """
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
+from enterprise.models import EnterpriseCustomer
```

Comment:
As long as we're not making it harder for ourselves in the future I'm okay with it -- I think the scope here is fairly contained, especially with the recent decision to combine everything into the same package.